### PR TITLE
update about page manifesto section to use reduced padding icons (fix #15679)

### LINKED ISSUE #15679: Left align the picto icons on the about page
Screenshot 2024-12-03 at 10 52 00 AM - Jodie Gatlin

This is going to be a PITA. I'm not sure there's a "right" way to do it that isn't a hack.

### PR DESCRIPTION
**Pull request by wen-2018**
## One-line summary

This PR updates about page hero section to use reduced padding icons to better match the designs

## Significant changes and points to review

- About page hero icons
- Check both `ltr` and `rtl`
- Check various screen sizes

## Issue / Bugzilla link

https://github.com/mozilla/bedrock/issues/15679

## Testing
http://localhost:8000/en-US/about/

### CODE REVIEW

File: media/img/icons/m24-reduced-padding/bell.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg width="14" height="15" fill="none" xmlns="http://www.w3.org/2000/svg"><g clip-path="url(#a)"><path d="M13.4957 10.4577v1.7408H7.671v-1.7408h2.724V5.84282c0-1.98618-1.61243-3.60203-3.59437-3.60203-1.98197 0-3.5944 1.61585-3.5944 3.60203v4.61488h2.72398v1.7408H.105469v-1.7408H1.46544V5.84282C1.46544 2.89678 3.8588.5 6.8006.5c2.94181 0 5.3352 2.39678 5.3352 5.34282v4.61488h1.3599ZM7.671 12.1985H5.93021l-.00001 1.7408h1.74079l.00001-1.7408Z" fill="#000"/></g><defs><clipPath id="a"><path fill="#fff" transform="translate(.105469 .5)" d="M0 0h13.39v14H0z"/></clipPath></defs></svg>
```
Comment (maureenlholland) [inline media/img/icons/m24-reduced-padding/bell.svg:1]:
```suggestion
<svg width="14" height="15" viewBox="0 0 14 15" fill="none" xmlns="http://www.w3.org/2000/svg"><g clip-path="url(#a)"><path d="M13.4957 10.4577v1.7408H7.671v-1.7408h2.724V5.84282c0-1.98618-1.61243-3.60203-3.59437-3.60203-1.98197 0-3.5944 1.61585-3.5944 3.60203v4.61488h2.72398v1.7408H.105469v-1.7408H1.46544V5.84282C1.46544 2.89678 3.8588.5 6.8006.5c2.94181 0 5.3352 2.39678 5.3352 5.34282v4.61488h1.3599ZM7.671 12.1985H5.93021l-.00001 1.7408h1.74079l.00001-1.7408Z" fill="#000"/></g><defs><clipPath id="a"><path fill="#fff" transform="translate(.105469 .5)" d="M0 0h13.39v14H0z"/></clipPath></defs></svg>
```

File: media/css/m24/feature.scss [hunk 2]
```diff
@@ -60,8 +60,8 @@
     text-wrap-style: balance;
 
     .mzp-c-picto-image {
-        height: 48px;
-        width: 48px;
+        height: 24px;
```
Comment (maureenlholland) [inline media/css/m24/feature.scss:63]: **question**: was it also requested to change size on smaller screens?

**issue (potentially blocking)** vertical spacing seems to become bit tight on smaller screen, not sure if that was a design choice

prod
Screenshot 2024-12-17 at 11 14 51 AM

pr

Screenshot 2024-12-17 at 11 14 43 AM

**Review by maureenlholland — COMMENTED** [review]
Height is standardized at 15 ✅ 
Things are lining up a lot better on inline axis ✅ 

Suggest adding `viewBox` attribute to SVGs (I believe this helps preserve the aspect ratio in scaling)

Need a bit of clarification on the design expectation for smaller screen vertical spacing before official approval

File: media/css/m24/feature.scss (hunk 2 above)
Comment (wen-2018) [reply media/css/m24/feature.scss:63]: The padding was reduced, so the image size reduction is to make them closer to what we had with the padding icons. This icon size might reduced a bit much. Will increase them slightly and add the space back with extra padding

**Review by maureenlholland — APPROVED** [review]
LGTM r+ :shipit:

**Pull request merged**